xlators/cluster/ec/src/ec-helpers.c

```diff
@@ -734,6 +734,38 @@ ec_inode_get(inode_t *inode, xlator_t *xl)
     return ctx;
 }
 
+void
+ec_inode_readmask_set(inode_t *inode, xlator_t *xl, uintptr_t read_mask)
+{
+    ec_inode_t *ctx = NULL;
+
+    LOCK(&inode->lock);
+
+    ctx = __ec_inode_get(inode, xl);
+
+    ctx->read_mask = read_mask;
+
+    UNLOCK(&inode->lock);
+
+}
+
+uintptr_t
+ec_inode_readmask_get(inode_t *inode, xlator_t *xl)
+{
+    ec_inode_t *ctx = NULL;
+    uintptr_t read_mask;
+
+    LOCK(&inode->lock);
+
+    ctx = __ec_inode_get(inode, xl);
```

Comment:
It is not supposed to happen for a linked_inode but handle ctx being null, incase someone erroneously calls it on not yet linked inode.